suites/portal/studyViewerTest.js

```diff
@@ -1,46 +1,53 @@
-Feature('Study Viewer / Requestor');
+Feature('Study Viewer');
```

Comment:
we're missing a [test for multiple study viewers](https://github.com/uc-cdis/gen3-qa/blob/master/docs/testplans/study-viewer/study-viewer.md#tests-scenarios). i guess it could wait for when the study viewer gets deployed to the jenkins environments, but adding a reminder/placeholder in this file would be good

Reply:
i created ticket PXP-7215

Reply:
👍